src/textual/driver.py

```diff
@@ -18,7 +18,7 @@ class Driver(ABC):
     def __init__(self, console: "Console", target: "MessageTarget") -> None:
         self.console = console
         self._target = target
-        self._loop = asyncio.get_event_loop()
+        self._loop = asyncio.get_running_loop()
```

Comment:
now that `get_event_loop()` is deprecated we should replace it with `get_running_loop()` when possible